.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+trim_trailing_whitespace = true
+
+[*.java]
+indent_size = 4
+max_line_length = 100
```

Comment:
I think we have more to 120 for this.

Reply:
I took the numbers from here: https://github.com/apache/tomcat/blob/master/CONTRIBUTING.md#L127